clox/functions.py

```diff
@@ -208,20 +245,27 @@ def main():
     parser = argparse.ArgumentParser()
     parser.epilog = ADDITIONAL_INFO
     parser.add_argument('--timezone', help='timezone', type=str.upper, choices=TIMEZONES_LIST)
+    parser.add_argument('--country', help='country(iso3166 code)', type=str.upper, choices=COUNTRIES_LIST)
```

Comment:
`help='country (iso3166 code)'`

Reply:
Fixed in https://github.com/sepandhaghighi/clox/pull/42/commits/ab74e3c1a88a99151e481d4c4f4f51d87a1f0851